cirq-core/cirq/transformers/drop_negligible_operations_test.py

```diff
@@ -48,13 +48,12 @@ def test_clears_known_empties_even_at_zero_tolerance():
         cirq.Z(a) ** 0, cirq.Y(a) ** 0.0000001, cirq.X(a) ** -0.0000001, cirq.CZ(a, b) ** 0
     )
     cirq.testing.assert_same_circuits(
-        cirq.drop_negligible_operations(circuit, atol=0.001), cirq.Circuit([cirq.Moment()] * 4)
+        cirq.drop_negligible_operations(circuit, atol=0.001), cirq.Circuit([cirq.Moment()] * 3)
```

Comment:
what is happening here and in the qasm file?

Reply:
The differences are linked to the fact that now even exponents return the identity operator and thus "disappear" without any impact. For example in the QASM file before there were things like `rz(0) q[0];` that now are avoided.

(I now see that, after merging main, some tests stopped passing, I will look into those)